src/Badge.js

```diff
@@ -14,6 +14,10 @@ const type = props => {
       backgroundColor: props.theme.colors.lightBlue,
       color: props.theme.colors.darkBlue
     },
+    text: {
+      backgroundColor: props.theme.colors.text,
+      color: props.theme.colors.white
```

Comment:
Started to identify repeating code pattern here like `color: props.theme.colors.white` is being repeated by, for instance: `text` & `green`. Is it possible to avoid repeating `color` style code shared by different values of the`bg` prop support?

Reply:
Based on the discussion in #349 we've gone in a different direction!  Rather than adding more colors presets to type, we're going to allow any color combination.  `type` will be left in for now to support previous use cases.